Example/Emission/AppDelegate.m

```diff
@@ -294,7 +300,7 @@ - (UIViewController *)viewControllerForRoute:(NSString *)route;
       params[item.name] = item.value;
     }
     viewController = [[ARGeneComponentViewController alloc] initWithGeneID:geneID refineSettings:params];
-
+    
```

Comment:
Xcode has a handy setting to prevent this from happening:

<img width="912" alt="Screen Shot 2019-05-09 at 9 19 49 AM" src="https://user-images.githubusercontent.com/498212/57456520-aba70f00-723b-11e9-9361-6b0b87166c2a.png">